### PR TITLE
Don't allow toggling helm install flag

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: pkg/kotsutil/kots.go [hunk 1]
```diff
@@ -416,6 +416,42 @@ func LoadKotsKindsFromPath(fromDir string) (*KotsKinds, error) {
...
+			}
+
+			contents, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			decoded, gvk, err := decode(contents, nil, nil)
+			if err != nil {
+				return nil
```
Comment (emosbaugh) [inline pkg/kotsutil/kots.go:439]: wrap this with the path since its in a loop

File: pkg/kotsutil/kots.go [hunk 2]
```diff
@@ -416,6 +416,42 @@ func LoadKotsKindsFromPath(fromDir string) (*KotsKinds, error) {
...
+				return err
+			}
+
+			if info.IsDir() {
+				return nil
+			}
+
+			contents, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
```
Comment (emosbaugh) [inline pkg/kotsutil/kots.go:434]: wrap this with the path since its in a loop

File: pkg/pull/pull.go [hunk 3]
```diff
@@ -269,6 +275,24 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
...
+	}
+
+	for _, prevChart := range prevHelmCharts {
+		for _, newChart := range newHelmCharts {
+			if prevChart.Spec.Chart.Name != newChart.Spec.Chart.Name {
+				continue
+			}
+			if prevChart.Spec.UseCLIInstall != newChart.Spec.UseCLIInstall {
+				log.FinishSpinnerWithError()
+				return "", errors.Errorf("deployment method for chart %s has changed", newChart.Spec.Chart.Name)
```
Comment (emosbaugh) [inline pkg/pull/pull.go:291]: i thought you just cant change it from UseCLIInstall to ! UseCLIInstall

Comment (divolgin) [reply pkg/pull/pull.go:291]: Toggling is untested.  Future feature?

File: pkg/kotsutil/kots.go (hunk 1 above)
Comment (divolgin) [reply pkg/kotsutil/kots.go:439]: not an error

**Review by sgalsaleh — APPROVED** [review]

File: pkg/pull/pull.go (hunk 3 above)
Comment (emosbaugh) [reply pkg/pull/pull.go:291]: So this works on new installs only?

Comment (divolgin) [reply pkg/pull/pull.go:291]: Yes, that was the intention.

**Pull request merged**